package.json

```diff
@@ -36,10 +36,12 @@
     "fs-extra": "0.30.0",
     "image-size": "0.5.0",
     "jimp": "0.2.24",
+    "jsonpath": "0.2.6",
```

Comment:
Update CHANGES.md for this and `underscore`.  I also think we should evaluate this library a bit more as I mentioned in  #150.  It actually could be find if it is slower than custom functions given that the bottlenecks are elsewhere, but I would want to know is it just 2x slower?  50x slower?  How big is it?  etc.